src/SldStyleParser.ts

```diff
@@ -212,23 +212,23 @@ export class SldStyleParser implements StyleParser {
 
   /**
    * Get the GeoStyler-Style MarkSymbolizer from an SLD Symbolizer
-   * 
+   *
    * @param {object} sldSymbolizer The SLD Symbolizer
-   * @return {MarkSymbolizer} The GeoStyler-Style MarkSymbolizer 
+   * @return {MarkSymbolizer} The GeoStyler-Style MarkSymbolizer
    */
   getMarkSymbolizerFromSldSymbolizer(sldSymbolizer: any): MarkSymbolizer {
     const wellKnownName: string = _get(sldSymbolizer, 'Graphic[0].Mark[0].WellKnownName[0]');
     const strokeParams: any[] = _get(sldSymbolizer, 'Graphic[0].Mark[0].Stroke[0].CssParameter') || [];
     const opacity: string = _get(sldSymbolizer, 'Graphic[0].Opacity[0]');
     const radius: string = _get(sldSymbolizer, 'Graphic[0].Size[0]');
```

Comment:
Could you rename the variable `radius` to `size`? Otherwise it's kind of getting confusing when you set
```markSymbolizer.radius = parseFloat(radius)/2```.

Reply:
Done